xc7/make/device_define.cmake

```diff
@@ -169,19 +169,23 @@ function(ADD_XC7_DEVICE_DEFINE_TYPE)
 
   set(SDF_TIMING_DIRECTORY ${PRJXRAY_DB_DIR}/${ARCH}/timings)
   set(UPDATE_ARCH_TIMINGS ${symbiflow-arch-defs_SOURCE_DIR}/utils/update_arch_timings.py)
+  set(UPDATE_PACK_PATTERNS ${symbiflow-arch-defs_SOURCE_DIR}/xc7/utils/add_pack_patterns.py)
   set(PYTHON_SDF_TIMING_DIR ${symbiflow-arch-defs_SOURCE_DIR}/third_party/python-sdf-timing)
   set(BELS_MAP ${symbiflow-arch-defs_SOURCE_DIR}/xc7/bels.json)
 
+  set(ADD_PACK_PATTERN "${PYTHON3} ${UPDATE_PACK_PATTERNS} --in_arch /dev/stdin")
+  set(PACK_PATTERN_DEPS UPDATE_PACK_PATTERNS)
+
   set(TIMING_IMPORT "${PYTHON3} ${UPDATE_ARCH_TIMINGS} --sdf_dir ${SDF_TIMING_DIRECTORY} --bels_map ${BELS_MAP} --out_arch /dev/stdout --input_arch /dev/stdin")
-  set(TIMING_DEPS "")
+  set(TIMING_DEPS BELS_MAP UPDATE_ARCH_TIMINGS)
```

Comment:
I believe this should be the files themselves, not the variable names

Reply:
Oh sure, my mistake. Should `add add_file_target` I suppose

Reply:
`add_file_target` is not required here, as these aren't generated files, and the dependency list goes directly to the add_custom_command

Reply:
Done